README.md

```diff
@@ -10,7 +10,7 @@ Thanos is a set of components that can be composed into a highly available metri
 
 Thanos is a [CNCF](https://www.cncf.io/) Incubating project.
 
-Thanos leverages the Prometheus 2.0 storage format to cost-efficiently store historical metric data in any object storage while retaining fast query latencies. Additionally, it provides a global query view across all Prometheus installations and can merge data from Prometheus HA pairs on the fly.
+Thanos leverages the Prometheus 2.0 storage format to cost-efficiently store historical metric data in any object storage while retaining fast query latencies. Additionally, it provides a global query view across all Prometheus installations and can merge data from the Prometheus HA pairs on the fly.
```

Comment:
This is not needed IMO. `the` implies a specific pair of Prometheus servers, when we are actually referring to pairs of servers in general.